src/main/java/dev/j3fftw/worldeditslimefun/commands/WorldEditSlimefunCommands.java

```diff
@@ -1,115 +1,187 @@
 import io.github.thebusybiscuit.slimefun4.api.items.SlimefunItem;
 import io.github.thebusybiscuit.slimefun4.core.handlers.BlockBreakHandler;
 import io.github.thebusybiscuit.slimefun4.core.handlers.BlockPlaceHandler;
 import io.github.thebusybiscuit.slimefun4.implementation.items.blocks.UnplaceableBlock;
 import io.github.thebusybiscuit.slimefun4.libraries.dough.blocks.BlockPosition;
 import me.mrCookieSlime.Slimefun.api.BlockStorage;
 import org.bukkit.ChatColor;
-import org.bukkit.Location;
 import org.bukkit.Material;
 import org.bukkit.block.Block;
 import org.bukkit.block.BlockFace;
 import org.bukkit.entity.Player;
 import org.bukkit.event.block.BlockBreakEvent;
 import org.bukkit.event.block.BlockPlaceEvent;
+import org.bukkit.inventory.EquipmentSlot;
+import org.bukkit.inventory.ItemStack;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Consumer;
+
+@SuppressWarnings("unused")
 @CommandAlias("wesf|sfedit")
+@CommandPermission("wesf.admin")
 public class WorldEditSlimefunCommands extends BaseCommand {
 
+    public static void init(WorldEditSlimefun plugin) {
+        PaperCommandManager manager = new PaperCommandManager(plugin);
+        CommandCompletions<BukkitCommandCompletionContext> completions = manager.getCommandCompletions();
+        CommandContexts<BukkitCommandExecutionContext> contexts = manager.getCommandContexts();
+
+        completions.registerStaticCompletion("slimefun_blocks", Utils.SLIMEFUN_BLOCKS);
+        completions.registerAsyncCompletion("command_flags", context -> {
+            List<String> args = new ArrayList<>(Arrays.asList(context.getContextValueByName(String[].class, "commandFlags")));
+            List<String> availableFlags = new ArrayList<>(CommandFlags.FLAG_TYPES.keySet());
+            availableFlags.removeAll(args);
+
+            if (args.isEmpty()) {
+                return availableFlags;
+            }
+
+            String currentArg = args.remove(args.size() - 1);
+            if (args.isEmpty()) {
+                return availableFlags;
+            }
+
+            String lastArg = args.get(args.size() - 1);
+            if (CommandFlags.FLAG_TYPES.containsKey(lastArg)) {
+                return CommandFlags.FLAG_TYPES.get(lastArg).getTabSuggestions(context);
+            }
+
+            if (args.size() % 2 == 0) {
+                return availableFlags;
+            }
+
+            return List.of("invalid_flag");
+        });
+
+        manager.registerCommand(new WorldEditSlimefunCommands());
+    }
+
     @Default
     public void onDefault(Player player) {
         player.sendMessage(ChatColor.RED + "Please provide a valid subcommand.");
     }
 
+    @Subcommand("wand")
+    public void onWand(Player player) {
+        ItemStack wand = SlimefunItem.getOptionalById("WESF_WAND").map(SlimefunItem::getItem).orElse(null);
+        if (wand == null) {
+            player.sendMessage(ChatColor.RED + "Wand not found!");
```

Comment:
Will this ever be hit?
if not please place a comment that this shouldnt ever be able to hit